packages/echo/src/handler.ts

```diff
@@ -64,9 +66,10 @@ export class EchoRequestHandler<Input extends any[] = any[], Output = any> {
 
   constructor(options: IEchoRequestHandlerOptions<Input, Output>) {
     this.handler = options.handler;
-    this.client = options.client;
+    this.client = options.client ? options.client : new Echo();
+    this.client.addWorkflows(options.workflow);
```

Comment:
@rifont do we concern ourselves that devs might use this method as well? We don't say it it in our docs (note to self: update readme with how to use the framework) but there is still a chance devs might see this and use it

Reply:
if I make it private then it is not accessible in the handler either. I can move all the logic associated with persisting the workflows in the handler but it is more work.

Is there a way in TS to make a class method available within the package but not outside of it?

Reply:
let's not worry about that now. The docs talk about how we want to use it so if a developer uses the other methods without understanding how it works it is on them. We have provided a good DX, but we cannot force peoples hand. We can circle back to this if it becomes a problem

Reply:
With the switch to implicitly instantiate the client, I don't believe most Developers will see the client methods anymore.

We could take a good variety of approaches to not expose it, but I also like the idea of the Framework providing escape hatches for Developers to build custom things on top of. So long as we choose appropriate naming, which I believe `addWorkflows` is, we should grant Developers flexibility to go wild with this. We might learn something from their creativity that we can build out as a first-class capability later.